sp/models.py

```diff
@@ -44,6 +44,11 @@ class IdP(models.Model):
     )
     contact_name = models.CharField(max_length=100)
     contact_email = models.EmailField(max_length=100)
+
+    authn_requests_signed = models.BooleanField(
```

Comment:
Minor quibble, but can you put this down by the `logout_request_signed` and `logout_response_signed` fields?